src/main/java/org/prebid/server/deals/targeting/RequestContext.java

```diff
@@ -269,6 +284,18 @@ private static Integer nodeToInteger(JsonNode node) {
         return node.isInt() ? node.asInt() : null;
     }
 
+    private static <T> List<T> nodeToListOfStrings(JsonNode node, Function<JsonNode, T> valueExtractor) {
```

Comment:
Why not `private static List<String> nodeToListOfStrings(JsonNode node, Function<JsonNode, String> valueExtractor)`, I mean why do we use generic return type, when method name implies that return type should be `List<String>`?

Reply:
Agreed.